src/Sulu/Component/Rest/RestController.php

```diff
@@ -11,18 +11,36 @@
 
 namespace Sulu\Component\Rest;
 
-use FOS\RestBundle\Controller\FOSRestController;
+use FOS\RestBundle\Controller\ControllerTrait;
 use FOS\RestBundle\View\View;
+use FOS\RestBundle\View\ViewHandlerInterface;
 use Sulu\Bundle\CoreBundle\Entity\ApiEntity;
 use Sulu\Component\Rest\Exception\EntityNotFoundException;
 use Sulu\Component\Rest\Exception\RestException;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
 
 /**
  * Abstract Controller for extracting some required rest functionality.
  */
-abstract class RestController extends FOSRestController
+abstract class RestController
```

Comment:
we should not change this class because most of our bundles extend this too. a better solution would be deprecating this class and add a new non-deprecated base class for the refactored controllers.